### PR TITLE
Update images docs

### PR DESCRIPTION
**Pull request by hakman**
Preview URL: https://deploy-preview-9736--kubernetes-kops.netlify.app/operations/images/

Fixes https://github.com/kubernetes/kops/issues/9727.

### CODE REVIEW

File: docs/operations/images.md [hunk 1]
```diff
@@ -119,54 +135,33 @@ aws ec2 describe-images --region us-east-1 --output table \
...
+  --filters "Name=name,Values=Flatcar-stable-*-hvm"
 ```
 
-## CentOS 7
-
-CentOS 7 is based on Kernel version **3.10** which has a considerable number of known bugs that affect it and may be noticed in production clusters:
-
-* [kubernetes/kubernetes#56903](https://github.com/kubernetes/kubernetes/issues/56903)
-* [kubernetes/kubernetes#67577](https://github.com/kubernetes/kubernetes/issues/67577)
+### Kope.io
```
Comment (johngmyers) [inline docs/operations/images.md:151]: Should go into the "Deprecated Distros" section.

Comment (hakman) [reply docs/operations/images.md]: True. Done.
I think, based on the Debian 9 moving to LTS, we should deprecate it too soon (but not remove it). Just show people it's a bad idea to use it.

File: docs/operations/images.md [hunk 2]
```diff
@@ -202,44 +179,66 @@ aws ec2 describe-images --region us-east-1 --output table \
   --filters "Name=name,Values=RHEL-8.*x86_64*"
 ```
 
-## Amazon Linux 2
+### Ubuntu 18.04 (Bionic)
 
-Amazon Linux 2 is based on Kernel version **4.14** which fixes some of the bugs present in RHEL/CentOS 7 and effects are less visible, but it's still quite old.
+Ubuntu 18.04 is based on Kernel version **4.15** which has number of known bugs that affect it and may be noticed with larger clusters:
```
Comment (johngmyers) [inline docs/operations/images.md:184]:
```suggestion
Ubuntu 18.04 is based on Kernel version **4.15** which has a number of known bugs that affect it and which may be noticed with larger clusters:
```

File: docs/operations/images.md [hunk 3]
```diff
@@ -202,44 +179,66 @@ aws ec2 describe-images --region us-east-1 --output table \
...
+### CoreOS
+
+Support for CoreOS is deprecated and was removed in _kops_ 1.18.
 
 You should consider using [Flatcar](#flatcar) as a replacement.
 
-## Debian 8 (Jessie)
+### Debian 8 (Jessie)
+
+Support for Debian 8 (Jessie) is deprecated and was removed in _kops_ 1.18.
```
Comment (johngmyers) [inline docs/operations/images.md:221]:
```suggestion
Support for Debian 8 (Jessie) was removed in _kops_ 1.18.
```

File: docs/operations/images.md [hunk 4]
```diff
@@ -202,44 +179,66 @@ aws ec2 describe-images --region us-east-1 --output table \
...
+  --filters "Name=name,Values=ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-*"
 ```
 
-## CoreOS
+## Deprecated Distros
 
-Support for CoreOS is deprecated and will be removed in Kops 1.18.
+### CoreOS
+
+Support for CoreOS is deprecated and was removed in _kops_ 1.18.
```
Comment (johngmyers) [inline docs/operations/images.md:215]:
```suggestion
Support for CoreOS was removed in _kops_ 1.18.
```

File: docs/operations/images.md [hunk 5]
```diff
@@ -1,51 +1,37 @@
...
-* `amazon` => `137112412989`
-* `centos` => `679593333241`
-* `debian9` => `379101102735`
-* `debian10` => `136693071363`
-* `flatcar` => `075585003325`
-* `redhat` => `309956199498`
-* `ubuntu` => `099720109477`
-
-For example, to use Ubuntu 20.04, you could specify:
+Using the AMI id is precise, but ids vary by region. It is often more convenient to use the `<owner/alias>/<name>`, if equivalent images have been copied to various regions with the same name.
```
Comment (johngmyers) [inline docs/operations/images.md:11]:
```suggestion
Using the AMI id is precise, but ids vary by region. It is often more convenient to use the `<owner/alias>/<name>` if equivalent images with the same name have been copied to other regions. 
```

File: docs/operations/images.md [hunk 6]
```diff
@@ -1,51 +1,37 @@
 # Images
 
-The default images for Kops clusters are based on Debian 9 (Stretch). These images are built by the [kubernetes-sigs/image-builder](https://github.com/kubernetes-sigs/image-builder/tree/master/images/kube-deploy/imagebuilder) project for each Kubernetes release and are available for any other use under the `kope.io` account.
+Starting Kubernetes 1.18, the default images used by _kops_ are the **[official Ubuntu 20.04](#ubuntu-2004-focal)** images.
```
Comment (johngmyers) [inline docs/operations/images.md:3]:
```suggestion
As of Kubernetes 1.18 the default images used by _kops_ are the **[official Ubuntu 20.04](#ubuntu-2004-focal)** images.
```

**Pull request merged**